composer.json

```diff
@@ -44,7 +44,7 @@
 		"horde/text-flowed": "dev-patch-1 as 2.0.3",
 		"horde/util": "dev-patch-1 as 2.5.8",
 		"ircmaxell/random-lib": "^1.2",
-		"league/commonmark": "^1.3",
+		"league/commonmark": "^1.4",
```

Comment:
Since you are using 1.4, you should change the code to use the non-deprecated classes.

Reply:
@GrahamCampbell thanks for noticing.

This is still WIP (GitHub doesn't allow on/off WIP state, like GitLab and I forgot on initial PR creation)

Reply:
> GitHub doesn't allow on/off WIP state

It does actually. They launched this a couple of weeks ago. ;)

Reply:
@GrahamCampbell: oh.